src/components/TopBar/Notifications/Notifications.styles.ts

```diff
@@ -0,0 +1,48 @@
+import { styled } from '@mui/system';
+import { Box, Button } from '@mui/material';
+
+import theme from 'styles/theme';
+
+import { ACCOUNT_HEADER_HEIGHT, ACCOUNT_FOOTER_HEIGHT } from '../AccountPanel/AccountPanel.const';
+import { NOTIFICATIONS_HEADER_HEIGHT } from './Notifications.const';
+
+export const StyledHeader = styled(Box)`
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  height: ${NOTIFICATIONS_HEADER_HEIGHT};
+`;
+
+export const StyledHeaderLeft = styled(Box)`
+  display: flex;
+  align-items: center;
+  margin-right: ${theme.spacing(1)};
+`;
+
+export const StyledHeaderRight = styled(Box)`
+  display: flex;
+  align-items: center;
+`;
+
+export const StyledIconWrapper = styled(Box)`
+  display: flex;
+  align-items: center;
+  margin-right: ${theme.spacing(0.8)};
+`;
+
+export const StyledCollapseBtn = styled(Button)`
+  display: flex;
+  align-items: center;
+  padding: 0;
```

Comment:
Better to move padding  0 to global theme

Reply:
It may affect all other buttons which always have the same paddings?

Reply:
We have many padding 0  for  button, we should do something with this

Reply:
then maybe create a new custom variant for that?

Reply:
done